typings/schema.pyi

```diff
@@ -1,17 +1,23 @@
 """
 This type stub file was generated by pyright.
 """
-from typing import Any, Callable, Dict, Generic, List
-from typing import Literal as LiteralTyping
-from typing import Optional as OptionalTyping
-from typing import Protocol, Tuple, TypeVar, Union, overload
+from types import BuiltinFunctionType, FunctionType
+from typing import Any, Callable, Dict, List
+from typing import Literal as LiteralType
+from typing import Optional as OptionalType
+from typing import Tuple, Union, overload
+
+GenericType = Union["Schema", "And", "Or", "Use", "Optional", "Regex", "Literal", "Const"]
+AcceptedDictType = Dict[Union[str, GenericType], Any]
+_CallableLike = Union[FunctionType, BuiltinFunctionType, Callable[..., Any]]
```

Comment:
`Callable` should already include the above. Any exceptions that requires this

Reply:
@aarnphm

Reply:
Hmm I don't think so

Reply:
Any exception?